client/src/App.js

```diff
@@ -52,7 +52,11 @@ function App() {
               )}
             </Route>
             <Route exact path="/voting">
-              {isLoggedIn ? <Voting /> : <Redirect to="/NotLoggedIn" />}
+              {isLoggedIn ? (
+                <Voting onLogout={handleLogout} />
+              ) : (
+                <Redirect to="/NotLoggedIn" />
```

Comment:
The naming convention of your routes is unclear. `NotLoggedIn`, `editPassword`.

I would take a look at pages like GitHub to get examples for common route names like `/login`, `/forget-password`.